setup.sh

```diff
@@ -0,0 +1,31 @@
+#Step 1: sets the output if the script fails for error
+shebang | -ex
```

Comment:
You'll need a real shebang here:
```suggestion
#! /usr/bin/env bash -ex
```

Reply:
Resolved by including the bash